notebooks/Notebook2.md

```diff
@@ -18,17 +20,42 @@ exciting milestones, preliminary results, etc. Feel free to include images
 code, and any other resources that you think will help clearly convey your
 design process.
 
+Exciting milestones: I actually met all the "base" milestones I wanted to meet for the project. I created UnorderedString and PhonemicString. I implemented several operators, such as - subset, superset, minus, genral string operations, equals and equalsdistinct for UnorderedString -- AND toXcase, toIPACase, =~, length for PhonemicString. I also implemented the loop. Last week, I only had a barebones structure for only PhonemicString and the loop - and now I have actually functioning strings with several implemented functions, operators and flow control structures for two of the three types. 
+
+Beyond that, as mentioned, I also created a spreadsheet which specifies the three distinct types, and the functionality associated with each one.
+
+![image](https://user-images.githubusercontent.com/79538073/230794237-7f391bd4-6353-43d8-aa84-9240bcaecdcb.png)
```

Comment:
That was smart to add description for all the operators and functions because you can also use this spreadsheet to make the documentation for the language, explaining everything it can do. That is exciting you were able to complete everything you planned on and more.